src/AirBench/App_Start/WebApiConfig.cs

```diff
@@ -16,7 +13,7 @@ public static void Register(HttpConfiguration config)
 
             config.Routes.MapHttpRoute(
                 name: "DefaultApi",
-                routeTemplate: "api/{controller}/{id}",
+                routeTemplate: "api/{controller}/{action}/{id}",
```

Comment:
Why did you add an `action` route parameter? Are you actually using it? Feels like a violation of REST API design principles.

Reply:
I added the action parameter because I wanted to avoid using the RoutePrefix and the Route attributes like Curtis had us do for the DeckOfCards API. I consulted this to design my route actions: 

https://docs.microsoft.com/en-us/aspnet/web-api/overview/web-api-routing-and-actions/routing-in-aspnet-web-api#routing-by-action-name

I will refactor my API to better conform with RESTful principles.